content/200-concepts/150-data-platform/08-data-proxy.mdx

```diff
@@ -94,7 +94,15 @@ To generate Prisma Client with the Data Proxy, call `prisma generate` with the o
 npx prisma generate --data-proxy
 ```
 
-When you use `--data-proxy`, you get a very lightweight Prisma Client runtime of approximately 250kb. This is because the Data Proxy does not need local engine files, so the generation process does not include the [query engine](/concepts/components/prisma-engines/query-engine).
+Alternatively, you can set the environment variable `PRISMA_GENERATE_DATAPROXY=true` to generate a Prisma Client with Data Proxy when a normal `prisma generate` command is executed.
+
+<Admonition type="warning">
+
+When deploying to [Vercel Serverless Functions](https://vercel.com/docs/concepts/functions/serverless-functions) you _need_ to define `PRISMA_GENERATE_DATAPROXY=true` [as an environment variable in your Vercel project settings](https://vercel.com/docs/concepts/projects/environment-variables) for the correct Prisma Client for Data Proxy to be generated.
```

Comment:
"Prisma Client for Data Proxy" is acutally what we call the... 🥁🥁🥁  Prisma Client for Data Proxy usage. See https://github.com/prisma/prisma/releases/tag/3.15.0 for example 😆 

> ... the correct Prisma Client for the Data Proxy to use.

This would be not 100% correct unfortunately. The Data Proxy does not use Prisma Client. It is the other way around, the Prisma Client uses the Data Proxy to talk to the database.

We could maybe say "in order to generated the correct Prisma CLient to use the Data Proxy."?

Reply:
@Jan did this one make it in?

Reply:
Sure this works: "We could maybe say "in order to generate the correct Prisma Client to use the Data Proxy."?

Cause the current wording makes it sound like it is some sort of special Client, just for the data proxy.

Reply:
Accepted the separate suggestion.

But: We have the term `Prisma Client for Data Proxy` in release notes and many other places. So if you want to get rid of that, that is a lot more work.